configuration/openmrs/apps/clinical/dashboard.json

```diff
@@ -559,5 +559,16 @@
         "numberOfVisits": 5
       }
     }
-  }
+  },
+  "followup": {
```

Comment:
This should be a dashboard widget not a new dashboard

Reply:
Done